src/esp/bindings_js/bindings_js.cpp

```diff
@@ -276,7 +261,7 @@ EMSCRIPTEN_BINDINGS(habitat_sim_bindings_js) {
       .function("setState", &Agent::setState)
       .function("hasAction", &Agent::hasAction)
       .function("act", &Agent::act)
-      .function("getSubtreeSensors", &Agent::getSubtreeSensors);
+      .function("getSubtreeSensors", &Agent::jsGetSubtreeSensors);
```

Comment:
You can bind to a free function (a static global function defined in this file) instead of a Agent member method.

Reply:
Thanks for the clarification Eric! That's been addressed and is tested and working.